From 104342c599524fb16a28049ecf38cbdb41f3df55 Mon Sep 17 00:00:00 2001
From: FUKAUMI Naoki <naoki@radxa.com>
Date: Thu, 15 Sep 2022 03:48:06 +0000
Subject: [PATCH] arm64: dts: amlogic: fix interrupt storm from fusb302 on
 Radxa Zero

it makes load average >1. use correct pin for interrupt from fusb302.

Signed-off-by: FUKAUMI Naoki <naoki@radxa.com>
---
 arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts b/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts
index 39c2ee196e95e..4bf1eae38c5f9 100644
--- a/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts
@@ -340,7 +340,7 @@
 		pinctrl-0 = <&fusb302_irq_pins>;
 		pinctrl-names = "default";
 		interrupt-parent = <&gpio_intc>;
-		interrupts = <59 IRQ_TYPE_LEVEL_LOW>;
+		interrupts = <5 IRQ_TYPE_LEVEL_LOW>;
 
 		vbus-supply = <&typec2_vbus>;
 
